packages/core/.mocharc.js

```diff
@@ -29,6 +29,6 @@ if (!process.argv.includes("--e2e")) {
     extension: ["js"],
     watchExtensions: ["js"],
     spec: endToEndTests, // mocha will run this sent of test files defined in this array.
-    timeout: 80000,
+    timeout: 100000,
```

Comment:
Why this change?

Reply:
It's not actually required really. 80k would be fine as well. I changed it while having some issues with timeout on the bridge but the diff between 80k and 100k is really important. so no real reason to have change it

Reply:
nit: Ok I"m not even sure what this value does maybe add a small comment.